src/main/java/gregicadditions/jei/GAMultiblockInfoCategory.java

```diff
@@ -22,10 +25,17 @@ public GAMultiblockInfoCategory(IJeiHelpers helpers) {
 		this.background = guiHelper.createBlankDrawable(176, 166);
 	}
 
-	public static void registerRecipes(IModRegistry registry) {
-		registry.addRecipes(Lists.newArrayList(new MultiblockInfoRecipeWrapper(new AssemblyLineInfo()), new MultiblockInfoRecipeWrapper(new FusionReactor1Info()), new MultiblockInfoRecipeWrapper(new FusionReactor2Info()), new MultiblockInfoRecipeWrapper(new FusionReactor3Info()), new MultiblockInfoRecipeWrapper(new ProcessingArrayInfo())
+	public static final Map<String, MultiblockInfoRecipeWrapper> multiblockRecipes = new HashMap<String, MultiblockInfoRecipeWrapper>() {{
```

Comment:
We should avoid using the "trick" of double-brace initialization, as it has various negative qualities. Also of note is that `final` in this instance just means that the `Map` the variable `multiblockRecipes` points to cannot be changed. The map itself is completely mutable by any part of the code. That seems a bit sketchy in terms of effectively producing mutable global state.